src/lib.rs

```diff
@@ -60,6 +60,23 @@ fn backwards_compat_path(mut path: String) -> String {
     return path;
 }
 
+///
+/// Backwards compatibility convertor for RedisJSON 1.x clients
+///
+fn backwards_compat_path_clone(path: String) -> PathBackward {
```

Comment:
Now that we have a specific `PathBackward` struct for this, I would move this standalone function into a `new` constructor on that struct.

Reply:
done